Update templates

sven1103 opened the pull request:
This PR makes some simple updates to match the current requirements.

sven1103 committed:
Refactor some templates

KochTobi commented on qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/src/main/groovy.life.qbic/example/Example.groovy:
```diff
@@ -0,0 +1,5 @@
+package life.qbic.example
```
The folder structure of the project is wrong. There should not be one folder named `groovy.life.qbic` but three folders/packages `gorrvy` `life` and `qbic`

sven1103 replied:
damn, that was the IDE formatting it like this ...

KochTobi replied:
> If you want to create a number of nested directories, specify the directory names separated with slashes.

sven1103 committed:
Provide correct package structure

KochTobi committed:
Fix folder structure

This commit replaces a folder with the name `groovy.life.qbic` with `groovy`/`life`/`qbic`

KochTobi committed:
Merge branch 'feature/template-refactoring' of https://github.com/qbicsoftware/qube-cli into feature/template-refactoring

KochTobi commented:
The ServiceIntegrationTests still try to use Junit. Should this be changed to Spock also?

sven1103 committed:
Merge branch 'development' into feature/template-refactoring

sven1103 commented:
@KochTobi If you could rewrite it and open a new PR?

Zethson commented on qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/src/main/groovy/life/qbic/example/Example.groovy:
```diff
@@ -0,0 +1,5 @@
+package life.qbic.example
```
That's a groovy file inside a Java template.

lib_java has no groovy support (yet!).

sven1103 replied:
sure it has, via the parent pom ;)

Zethson replied:
Ahh, it's part of the parent pom.

In my (old) QBiC projects I included it manually :)

Still, for consistency I would prefer a .java here, but the choice is yours :)

KochTobi replied:
my IDE does not recognize that groovy is allowed/ included in the projects.

Zethson reviewed (approved):
LGTM besides .groovy vs .java file name

KochTobi commented on qube/create/templates/portlet/portlet_groovy/{{ cookiecutter.project_slug }}/pom.xml:
```diff
@@ -28,7 +28,7 @@
 			<id>nexus-snapshots</id>
 			<name>QBiC Snapshots</name>
-			<url>https://qbic-repo.am10.uni-tuebingen.de/repository/maven-snapshots</url>
+			<url>http://qbic-repo.qbic.uni-tuebingen.de:8081/repository/maven-snapshots</url>
```
Do we need the port here?
Did this change from `https:` to `http`?

KochTobi commented on qube/create/templates/portlet/portlet_groovy/{{ cookiecutter.project_slug }}/pom.xml:
```diff
@@ -41,44 +41,39 @@
 			<id>nexus-releases</id>
 			<name>QBiC Releases</name>
-			<url>https://qbic-repo.am10.uni-tuebingen.de/repository/maven-releases</url>
+			<url>http://qbic-repo.qbic.uni-tuebingen.de:8081/repository/maven-releases</url>
```
Do we need the port here?
Did this change from `https:` to `http`?

KochTobi commented on qube/create/templates/service/service_java/{{ cookiecutter.project_slug }}/pom.xml:
```diff
@@ -29,7 +28,7 @@
 			<id>nexus-snapshots</id>
 			<name>QBiC Snapshots</name>
-			<url>https://qbic-repo.am10.uni-tuebingen.de/repository/maven-snapshots</url>
+			<url>http://qbic-repo.qbic.uni-tuebingen.de:8081/repository/maven-snapshots</url>
```
Do we need the port here?
Did this change from `https:` to `http`?

KochTobi commented on qube/create/templates/service/service_java/{{ cookiecutter.project_slug }}/pom.xml:
```diff
@@ -42,18 +41,10 @@
 			<id>nexus-releases</id>
 			<name>QBiC Releases</name>
-			<url>https://qbic-repo.am10.uni-tuebingen.de/repository/maven-releases</url>
+			<url>http://qbic-repo.qbic.uni-tuebingen.de:8081/repository/maven-releases</url>
```
Do we need the port here?
Did this change from `https:` to `http`?

KochTobi commented on qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/pom.xml:
```diff
@@ -29,7 +28,7 @@
 			<id>nexus-snapshots</id>
 			<name>QBiC Snapshots</name>
-			<url>https://qbic-repo.am10.uni-tuebingen.de/repository/maven-snapshots</url>
+			<url>http://qbic-repo.qbic.uni-tuebingen.de:8081/repository/maven-snapshots</url>
```
Do we need the port here?
Did this change from `https:` to `http`?

KochTobi commented:
The portlet folder structure is also not correct. There is a folder with the name `life.qbic.portal.portlet`

Zethson committed:
Merge branch 'development' into feature/template-refactoring
